batch/oak_d_w.json

```diff
@@ -5,6 +5,12 @@
     "test_type": "OAK-D",
     "options": {"bootloader": "header_usb", "imu": true},
     "variants": [
+        {
+            "title":"OAK-D W (DM9098 R7M2E7)",
+            "description": "OAK-D W based on DM9098 R7M2E7 board, with wide lens",
+            "eeprom":"eeprom/DM9098_R7M2E7_oak_d_w.json",
+            "board_config_file": "OAK-D-W.json"
```

Comment:
Should it be OAK-D-W-97 instead?

Reply:
Ok, yes, it was a misunderstanding, this should indeed be the OAK-D-W-97. However, the config for that board already exists. I'm waiting for Andy to clear it up.